Makefile

```diff
@@ -101,15 +101,16 @@ data/unicharset: $(ALL_BOXES)
 	merge_unicharsets data/$(START_MODEL)/$(START_MODEL).lstm-unicharset $(GROUND_TRUTH_DIR)/my.unicharset  "$@"
 else
 data/unicharset: $(ALL_BOXES)
-	unicharset_extractor --output_unicharset "$@" --norm_mode 1 "$(ALL_BOXES)"
+	unicharset_extractor --output_unicharset "$@" --norm_mode $(NORM_MODE) "$(ALL_BOXES)"
 endif
 
 $(ALL_BOXES): $(sort $(patsubst %.tif,%.box,$(wildcard $(GROUND_TRUTH_DIR)/*.tif)))
 	find $(GROUND_TRUTH_DIR) -name '*.box' -exec cat {} \; > "$@"
 
-$(GROUND_TRUTH_DIR)/%.box: $(GROUND_TRUTH_DIR)/%.tif $(GROUND_TRUTH_DIR)/%.gt.txt
-	python generate_line_box.py -i "$(GROUND_TRUTH_DIR)/$*.tif" -t "$(GROUND_TRUTH_DIR)/$*.gt.txt" > "$@"
+$(GROUND_TRUTH_DIR)/%.box: $(GROUND_TRUTH_DIR)/%.tif 
+	tesseract "$(GROUND_TRUTH_DIR)/$*.tif" "$(GROUND_TRUTH_DIR)/$*" -l $(START_MODEL) --psm 6 wordstrbox > "$@"
```

Comment:
This is a pretty new thing, right? Although it would be nice to get rid of `generate_line_box.py`, the dependency on `START_MODEL` is a problem for training from scratch as the Makefile is set up right now. Is the model actually used while generating the box files? If not, we could simply use an empty model or include the box generation in the `ifdef` construction above.

Reply:
Also, the parameter file is not installed if I use the (latest) tesseract version form github.

Reply:
It is maybe not even part of the repo:
```shell
kmw@barclay:~/built/tesseract$ find . -type f -name "*struct"
kmw@barclay:~/built/tesseract$ find . -type f -name "*struct*"
./src/cutil/.deps/structures.Plo
./src/cutil/.libs/structures.o
./src/cutil/structures.lo
./src/cutil/structures.o
./src/cutil/structures.cpp
./src/cutil/structures.h
./src/ccstruct/ccstruct.h
./src/ccstruct/libtesseract_ccstruct.la
./src/ccstruct/ccstruct.lo
./src/ccstruct/.deps/ccstruct.Plo
./src/ccstruct/.libs/libtesseract_ccstruct.a
./src/ccstruct/.libs/ccstruct.o
./src/ccstruct/ccstruct.o
./src/ccstruct/ccstruct.cpp
```

Reply:
Thank you for pointing out about the `wordstrbox` config file. I have submitted a PR to fix that. https://github.com/tesseract-ocr/tesseract/pull/2280

Reply:
I foresee additions to the makefile to handle the multiple options:

1. `generate_line_box.py` is useful for cases where the line images and ground truth is available.

2. `wordstrbox` is useful where scanned images are available and an easily editable box file is required for LSTM training. Please note that the makefile should not continue directly after these box files are generated because the box files will not be correct with the text.

The reason that a START_MODEL is required here is that the box files need to be generated with the best possible transcription for the scanned image. An empty START_MODEL by default uses `eng` - so it will not work for other languages.

Reply:
This was not supposed to be a replacement makefile but rather an example of how to use the new wordstrbox option.

Reply:
I see. The four options for training will then have to be reflected in a larger modification of the Makefile.

Reply:
>The four options for training will then have to be reflected in a larger modification of the Makefile.

Yes.

In the tesseract repo, tesstrain.sh is used to create synthetic training data - box/tiff pairs from fonts and training_text. That is the third option, in addition to the two I mentioned above.